### PR TITLE
Sidebar improvements

### PR DESCRIPTION
**Pull request by bobo-k2**
**Pull Request Summary**

Language selector styling is not perfect, since this is Quasar component.

| BEFORE | AFTER |
|---------|---------|
|image|image|

imageimage

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

### CODE REVIEW

File: src/components/common/LocaleChanger.vue [hunk 1]
```diff
@@ -1,47 +1,50 @@
...
+      },
+      { immediate: true }
+    );
+
+    watch(
+      [selectedLang],
+      () => {
+        locale.value = selectedLang.value.code;
         localStorage.setItem(LOCAL_STORAGE.SELECTED_LANGUAGE, locale.value);
+        console.log(selectedLang.value);
```
Comment (impelcrypto) [inline src/components/common/LocaleChanger.vue:42]: Do you need this line?

Comment (bobo-k2) [reply src/components/common/LocaleChanger.vue]: Of course not. Thank you. fixed

**Review by impelcrypto — APPROVED** [review]

**Comment by Kahonnohak** [comment]
@bobo-k2 thanks! Please take a look at few things. 

1, Dashboard disappeared. So I cannot click Asset page but instead "dashboard" links to Asset page and default is set to this.

2, NFT icon. Maybe my Figma sorry, but NFT had its paint pallet icon. please bring that back?

3, the bottom three buttons. 
- 3-1, Could you remove the hover styling please?
- 3-2, the meuu, I think hover is missing 

Thank you!

Screenshot 2023-07-14 at 07 25 12
Screenshot 2023-07-14 at 07 32 10

**Comment by bobo-k2** [comment]
> @bobo-k2 thanks! Please take a look at few things.
> 
Hey @Kahonnohak, thank you for comments. I believe I addressed them all. Please check again

**Review by Kahonnohak — APPROVED** [review]
Great Work!

**Pull request merged**